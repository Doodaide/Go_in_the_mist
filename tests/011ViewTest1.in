who
view
place E5
view
place K5
view
place G17
view
place S17
view
place P11
view
place B10
view
resign