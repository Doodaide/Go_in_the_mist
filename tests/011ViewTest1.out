B
J10,.................................................
L12,.................................................
P12,.................................................
E9,.................................................
E9,.................................................
C9,x......x......x......x......x......x......x......
L18,xxxxxxxxxxxxxx...................................
White wins!
E5K5G17S17P11B10
Thank you for playing!
